Tests/SwiftParserTest/VariadicGenericsTests.swift

```diff
@@ -10,7 +10,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-
+@_spi(RawSyntax) import SwiftSyntax
```

Comment:
I don’t think you should need `@_spi(RawSyntax)`. I might be wrong though.

Reply:
You're right, fixed!